src/utils/pr.test.ts

```diff
@@ -0,0 +1,36 @@
+import { describe, it, expect } from "vitest";
+import { getCheckboxValueFromString } from "./pr";
+
+describe("pr", () => {
+  it("should check for the correct checkbox value", () => {
+    const prDescription = `
+This PR was opened by the [ready-release-go](https://github.com/woodpecker-ci/plugin-ready-release-go) plugin.
+When you're ready to do a release, you can merge this and a release and tag with version
```

Comment:
```suggestion
When you're ready for the next release, you can merge this PR and a release with the respective version tag
```

Reply:
That's just a text for the test. There are some gaps as I removed the actual values while copying it from this repo.

Reply:
I've added some sample values for the gaps.